src/designer/elsa-workflows-studio/src/components/editors/properties/elsa-script-property/elsa-script-property.tsx

```diff
@@ -88,7 +133,26 @@ export class ElsaScriptProperty {
                      ref={el => this.monacoEditor = el}/>
       </div>
       {fieldHint ? <p class="elsa-mt-2 elsa-text-sm elsa-text-gray-500">{fieldHint}</p> : undefined}
+      {this.isJsonSchemaValid == undefined ? undefined : this.isJsonSchemaValid ?
+          <p class="elsa-mt-1 elsa-text-sm elsa-text-green-500">
+                Json is valid
+          </p>
+          :
+          <p class="elsa-mt-1 elsa-text-sm elsa-text-red-500">
+                Json is invalid
+          </p>
+      }
       <input type="hidden" name={fieldName} value={value}/>
+      {isSchema ?
+        <div class="elsa-mt-1">
+          <a href="#"
+              onClick={e => this.onConvertToJsonSchemaClick(e)}
+              class="elsa-relative elsa-inline-flex elsa-items-center elsa-px-4 elsa-py-2 elsa-border elsa-border-gray-300 elsa-text-sm elsa-leading-5 elsa-font-medium elsa-rounded-md elsa-text-gray-700 elsa-bg-white hover:elsa-text-gray-500 focus:elsa-outline-none focus:elsa-shadow-outline-blue focus:elsa-border-blue-300 active:elsa-bg-gray-100 active:elsa-text-gray-700 elsa-transition elsa-ease-in-out elsa-duration-150">
+              Convert to Json Schema
+          </a>
+        </div>
+        : undefined
+      }
```

Comment:
Keep in mind that this script property component is reused by other activities as wel, so this validation that is specific to the "Schema" property doesn't belong here.